docs/languages/python.md

```diff
@@ -0,0 +1,83 @@
+---
+Order: 8
+Area: languages
+TOCTitle: Python
+ContentId: c2cb770d-571d-4edf-9eb9-b5b8977c21a0
+PageTitle: Python with Visual Studio Code
+DateApproved: 6/2/2016
+MetaDescription: Learn about Visual Studio Code features (syntax highlighting, snippets, linting) for Python development.
+---
+
+# Python on Visual Studio Code
+
+Python is fully supported in Visual Studio Code through an extension. Popular extensions in the [Marketplace](https://marketplace.visualstudio.com) provide code completion, linting, debugging, code formatting, snippets, and more. 
+
+## Demo in 30 Seconds
+
+<video id="python-feature-showcase-video" src="/images/python_feature_showcase.mp4" poster="/images/python_feature_showcase_placeholder.png" autoplay loop controls muted></video>
+
+> Download - If you haven't downloaded Visual Studio Code yet, click [here](/download) to give it a try. 
+
+## Install Python Extension
+
+Visual Studio Code is a fast editor and ships with only the basic features. Customize Visual Studio Code with Python by installing one of the popular Python Extensions. 
+
+1. Select an extension. 
+2. Install the extension by typing `ext install` into the command palette `kb(workbench.action.showCommands)`. 
+
+<div class="marketplace-extensions-python"></div>
+
+> Tip: these extensions are dynamically queried. Read the description and reviews to decide which extension is best for you. See more in the [Marketplace](https://marketplace.visualstudio.com/vscode).
+
+The examples in this document will use [Don Jayamanne's Python Extension](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python) as it has the highest number of downloads and the most features. 
+
+## Code Completion
+
+[Intellisense](/docs/editor/editingevolved#_intellisense) is intelligent code completion (in-context method and variable suggestions) across all your files and for built-in and thirty-party modules. 
+
+Quickly see methods, class members, and documentation.
+
+<video id="python-code-completion-video" src="/images/python_python-intellisense.mp4" poster="/images/python_python-intellisense-placeholder.png" autoplay loop controls muted></video>
+
+> Tip: trigger code completion with `kb(editor.action.triggerSuggest)`.
```

Comment:
In general, capitalize the start of Tip and Note sentences.
Trigger code completion....